storage: fix liveness error string

tbg opened the pull request:
The old code would print

```
node_id:3 epoch:122 expiration:<wall_time:1497621451659892498 logical:0 > ; actual is {NodeID:3 Epoch:122 Expiration:1497621455.938854596,0 Draining
:false}
```

and the new code would use the `{NodeID: 3}` format for both.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/16561)
<!-- Reviewable:end -->

tbg committed:
storage: fix liveness error string

The old code would print

```
node_id:3 epoch:122 expiration:<wall_time:1497621451659892498 logical:0 > ; actual is {NodeID:3 Epoch:122 Expiration:1497621455.938854596,0 Draining
:false}
```

and the new code would use the `{NodeID: 3}` format for both.
